pygments/lexers/theorem.py

```diff
@@ -35,12 +35,12 @@ class CoqLexer(RegexLexer):
 
     keywords1 = (
         # Vernacular commands
-        'Section', 'Module', 'End', 'Require', 'Import', 'Export', 'Variable',
+        'Section', 'Module', 'End', 'Require', 'Import', 'Export', 'Include', 'Variable',
         'Variables', 'Parameter', 'Parameters', 'Axiom', 'Axioms', 'Hypothesis',
         'Hypotheses', 'Notation', 'Local', 'Tactic', 'Reserved', 'Scope',
-        'Open', 'Close', 'Bind', 'Delimit', 'Definition', 'Example', 'Let',
-        'Ltac', 'Fixpoint', 'CoFixpoint', 'Morphism', 'Relation', 'Implicit',
-        'Arguments', 'Types', 'Unset', 'Contextual', 'Strict', 'Prenex',
```

Comment:
Not sure why you're removing `Unset`, is it deprecated? (I don't see such information on https://coq.inria.fr/doc/V8.13.0/refman/language/core/basic.html#coq:cmd.Unset)

Reply:
Ah sorry, commented too quickly. I see it below.